docs/examples/blind-auction.rst

```diff
@@ -48,10 +48,21 @@ to receive their money - contracts cannot activate themselves.
         event HighestBidIncreased(address bidder, uint amount);
         event AuctionEnded(address winner, uint amount);
 
-        // The following is a so-called natspec comment,
-        // recognizable by the three slashes.
-        // It will be shown when the user is asked to
-        // confirm a transaction.
+        // Errors that describe failures.
+
+        // The triple-slash comments are so-called natspec
+        // comments. They will be shown when the user
+        // is asked to confirm a transaction or
+        // when an error is displayed.
+
+        /// The action has already ended.
+        error AuctionAlreadyEnded();
+        /// There is already a higher bid.
+        error BidNotHighEnough(uint highestBid);
+        /// The action has not ended yet.
+        error AuctionNotYetEnded();
+        /// The function actionEnd has already been called.
+        error AuctionEndAlreadyCalled();
```

Comment:
```suggestion
        /// The auction has already ended.
        error AuctionAlreadyEnded();
        /// There is already a higher bid.
        error BidNotHighEnough(uint highestBid);
        /// The auction has not ended yet.
        error AuctionNotYetEnded();
        /// The function auctionEnd has already been called.
        error AuctionEndAlreadyCalled();
```

Reply:
I see this is fixed now except for `actionEnd` (which I think should be `auctionEnd()`.